CONTRIBUTING.md

```diff
@@ -50,9 +50,10 @@ The M1 ("Apple silicon") is an Arm64 processor. While .NET 6 runs natively on th
     chmod +x ./dotnet-install.sh
     sudo ./dotnet-install.sh --version 2.1.818 --arch x64 --install-dir /usr/local/share/dotnet/x64
      ```
-- If using JetBrains Rider, launch it, go to `Preferences` -> `Build, Execution, Deployment` -> `Toolset and Build`, and set the following:
-  - .NET Core CLI executable path: `/usr/local/share/dotnet/dotnet` (*not* x64)
-  - Use MSBuild version: `17.0 - /usr/local/share/dotnet/sdk/6.0.201/MSBuild.dll` (or higher version)
+- If using JetBrains Rider, use it to launch `Sentry.sln` (or one of the `slnf` files).  Then go to `Preferences` -> `Build, Execution, Deployment` -> `Toolset and Build`, and set the following, which will tell Rider to use the x64 versions of the SDKs.  (This is *required* to run unit tests or debug using .NET 5.0 and earlier .NET Core SDKs.)
```

Comment:
is it `Preferences` or `Settings`? 
![image](https://user-images.githubusercontent.com/122666/161357253-9bbbdc65-d911-4eed-aa28-5aab3b0e474c.png)

Reply:
On macOS, it's `Preferences`.  Guess that's an OS-flavored thing.

<img width="313" alt="image" src="https://user-images.githubusercontent.com/1396388/161357349-662e5aa3-57e9-4e09-932f-5d5dd2f082b0.png">